### DISCUSSION TITLE
Lightweight lexical without prismjs

### DISCUSSION THREAD

**Discussion by RLowies2** [facebook/lexical#3778] [General]
Are there any plans to remove prismjs as a dependency? It adds a considerable size increase to the package.

**Comment by acywatson** [comment]
Not sure I'm following. The only package that uses prism is lexical-code. Unless you're using that (or the dependent CodeHighlightPluing in lexical-react), you're getting prism. It'd probably be good to make the code highlighting library agnostic, but if you want code highlighting, you need something like prism or you need to write something like it yourself.

**Comment by louisremi** [comment]
Note that using the Markdown plugin also results in prismjs being included in your code

**Reply by acywatson** [reply]
Good point - we should fix that, tbh.  Feel free to make an issue.

**Reply by canastro** [reply]
Is there any issue created that we can track? 

I'm currently facing some issues with prism even when I just want to use markdown but not supporting code blocks.

**Comment by evilmastermind** [comment]
Prismjs as a requirement for lexical-markdown also seems to cause issues in non-react enviroments (e.g. Vite)
